sonaranalyzer-dotnet/src/SonarAnalyzer.Common/Helpers/AnalysisContextExtensions.cs

```diff
@@ -58,6 +62,26 @@ private static void ReportWhenNotSuppressed(SyntaxTree tree, Diagnostic diagnost
         {
             if (SonarAnalysisContext.ShouldDiagnosticBeReported(tree, diagnostic))
             {
+                // VB.Net complier (VBC) post-process issues and will fail if the line contains the VbNetErrorPattern.
+                // See https://github.com/dotnet/roslyn/issues/5724
+                // As a workaround we will prevent reporting the issue if the issue is on a line with the error pattern and the
+                // language VB.Net
+                // TODO: Remove this workaround when issue is fixed on Microsoft side.

```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this 'TODO' comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=csharpsquid%3AS1135)